tests/acceptance/features/bootstrap/Sharing.php

```diff
@@ -44,17 +44,14 @@ trait Sharing {
 	 *
 	 * @var SimpleXMLElement[]
 	 */
-	private array $lastShareDataByUser = [];
+	private ?SimpleXMLElement $userSharesData = null;
```

Comment:
why both `createdUserShares` and `userSharesData`?

Reply:
`userSharesData` stores the response of the new created user share response where as `createdUserShare` stores all the important items from the response of user shares for all the shares created during a scenario.

Reply:
but do we need to store the response data? and for what purpose?

Reply:
that's a good point. I will update it.